tests/ashley/machina_extensions/forum_moderation/test_forum_moderation.py

```diff
@@ -182,3 +182,92 @@ def test_forum_moderation_list_forums_form(self):
             f"Select a valid choice. {forum3Lti2.id} is not one of the available choices.",
             html=True,
         )
+
+    def test_forum_moderation_list_forums_with_deleted_form(self):
+        """
+        Create three forums in the same LTIContext and archive one. Load the form to choose
+        to which forum the user wants to move the topic. Control that the forum deleted is
+        not part of the choice.
+        """
```

Comment:
add to the docstring that you also test that we get a 400 if the value is submitted.